compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -25,43 +25,91 @@
 namespace
 {
 
+/// @brief Returns the number of BroadcastTo(Builtin or Custom) node among cop's inputs
+int32_t num_of_broadcast_to(luci::CircleCustom *cop)
+{
+  int32_t bt_cnt = 0;
+  for (uint32_t idx = 0; idx < cop->numInputs(); idx++)
+  {
+    auto input = loco::must_cast<const luci::CircleNode *>(cop->inputs(idx));
+    switch (input->opcode())
+    {
+      case luci::CircleOpcode::BROADCAST_TO:
+      {
+        ++bt_cnt;
+        break;
+      }
+      case luci::CircleOpcode::CIRCLECUSTOMOUT:
+      {
+        auto inputOut = loco::must_cast<const luci::CircleCustomOut *>(cop->inputs(idx));
+        auto broadcastTo = loco::must_cast<luci::CircleCustom *>(inputOut->input());
+        if (broadcastTo->custom_code() == "BroadcastTo")
+          ++bt_cnt;
+        break;
+      }
+      default:
+        break;
+    }
+  }
+  return bt_cnt;
+}
+
 /// @brief Returns the index of BroadcastTo node among cop's inputs.
 // NOTE This function assumes there is only one BroadcastTo node among its inputs.
 int32_t get_broadcastTo_index_among_inputs_of(luci::CircleCustom *cop)
 {
-  int32_t bc_idx = -1;
+  if (num_of_broadcast_to(cop) != 1)
+    return -1;
+
   for (uint32_t idx = 0; idx < cop->numInputs(); idx++)
   {
     auto input = dynamic_cast<const luci::CircleCustomOut *>(cop->inputs(idx));
     if (input)
     {
       auto broadcastTo = loco::must_cast<luci::CircleCustom *>(input->input());
       if (broadcastTo->custom_code() == "BroadcastTo")
-      {
-        // check if there is only one BroadcastTo node
-        if (bc_idx != -1)
-        {
-          return -1;
-        }
-        bc_idx = static_cast<int32_t>(idx);
-      }
```

Comment:
@seanshpark 

> this part looks duplicate with get_broadcastTo_index_among_inputs_of.

I removed the part that duplicate with `num_of_broadcast_to` from `get_broadcastTo_index_among_inputs_of `, 
is that okay ?

Reply:
> ... is that okay ?

@mhs4670go PTAL

for me, I think there are multiple context of changes, it would be better to split this PR.

Reply:
Looks okay. Then, could you post a PR first that revises existing pass with `num_of_broadcast_to`?

Reply:
> Then, could you post a PR first that revises existing pass with num_of_broadcast_to?

yes! Yes. I will upload new PR first. :)

Reply:
@seanshpark @mhs4670go 

Another PR(https://github.com/Samsung/ONE/pull/12575) was merged first,
so Could you review this PR again? :)